spec/rails_helper.rb

```diff
@@ -41,6 +42,11 @@
 # If you are not using ActiveRecord, you can remove this line.
 ActiveRecord::Migration.maintain_test_schema!
 
+# Chrome
+WebMock.allow_net_connect!
+Webdrivers::Chromedriver.required_version = '2.46'
```

Comment:
I think this is having a conflict with travis. Travis is by default using the [latest stable version](https://docs.travis-ci.com/user/chrome#selecting-a-chrome-version) of chrome which is now at `81.0.4044.141` (https://chromereleases.googleblog.com/2020/05/stable-channel-update-for-chrome-os.html)

I'd remove this and see if the CI works